playbook/app/pb_kits/playbook/pb_rich_text_editor/TipTap/ToolbarDropdown.tsx

```diff
@@ -125,8 +130,10 @@ const popoverReference = (
             <Flex className={showPopover ? "fa-flip-vertical" : ""}
                 display="inline_flex"
             >
-              <Icon fixedWidth
-                  icon="angle-down"
+              <Icon 
+                  className="svg-inline--fa"
+                  customIcon={angleDown}
```

Comment:
Did we avoid just using `icon` here because we are trying to avoid using playbook-icons, and avoid having to ship playbook-icons as a part of playbook-ui? Maybe we could chat through some of this.